### PR TITLE
Adds section on parallelizing tests.

### PR DESCRIPTION
**Pull request by dbrumann**
See discussion in #11757

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Denis, thanks for contributing this content. I have a comment and a question.

Question: How many jobs are run in parallel? Is it configurable? Is it automatic based on the number of cores/threads?

Comment: I appreciate your detailed explanation ... but I'd prefer to remove all mentions to "parallel" and the example shown for it. I think it's out of the scope of the Symfony Docs and not adding that would simplify future maintenance. We could, however, mention explicitly that, when using PhpUnitBridge, you don't have to install any external tools, like "parallel", to achieve this. What do others think? @xabbuh?

Thanks!

**Comment by dbrumann** [comment]
That makes sense, I mainly put it there as a reference as this is how symfony/symfony parallelizes the suites and people who look at the `.travis.yml` might be interested to know what's happening in there. I agree though that it complicates the section a bit and is not really tied to the PhpUnitBridge.

As far as I can tell, there are is no way to configure how parallelization is done. The script collects all subfolders containing a `phpunit.xml.dist` and then loops over the results spawning a process for each one and then waits for all of them to finish; see: https://github.com/symfony/symfony/blob/4.4/src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php#L207-L222

**Comment by javiereguiluz** [comment]
Thanks for contributing these new contents and for rewording them during the review process.

File: components/phpunit_bridge.rst [hunk 1]
```diff
@@ -124,6 +124,32 @@ The summary includes:
...
+    │   │   └── phpunit.xml.dist
+
+.. code-block:: terminal
+
+    $ ./vendor/bin/simple-phpunit tests/
+
+The modified PHPUnit script will recursively go through the provided directory,
+up to a depth of 3 subfolders or the value specified by the environment variable
+``SYMFONY_PHPUNIT_MAX_DEPTH``, looking for ``phpunit.xml.dist`` files and then
+run each suite it finds in parallel, collecting their output and display each
```
Comment (OskarStark) [inline components/phpunit_bridge.rst]:
```suggestion
run each suite in parallel, collecting their output and display each
```

**Pull request merged**